lib/main.coffee

```diff
@@ -86,19 +86,25 @@ module.exports =
       type: 'boolean'
       default: true
       order: 13
+    spaceTriggersAutocomplete:
```

Comment:
Providers aren't available at this step in the process. What you're saying is that we should always trigger on space, but then let provider(s) determine if they should respond in a 'whitespace-only' scenario.